<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInformation</class>
 <widget class="QDialog" name="UserInformation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>新增用户</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0,0,0">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,7">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>姓  名:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_name">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,7">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>账  号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_username"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,7">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>部  门:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_department"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,7">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>权  限:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb_power"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,1,1">
     <item>
      <widget class="QPushButton" name="pb_reset">
       <property name="text">
        <string>重置密码</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ok">
       <property name="text">
        <string>确定</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_cancel">
       <property name="text">
        <string>取消</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
